astore/rpc/astore.proto

```diff
@@ -133,13 +133,23 @@ message NoteResponse {
   repeated Artifact artifact = 1;
 }
 
+message DeleteRequest {
+  string id = 1; //SID or UID (will be interpreted to which based on length)
```

Comment:
nit: make sure you run `go fmt` before submitting. We have no enforcement mechanism (yet, there should be one).